website/sidebars.json

```diff
@@ -116,7 +104,33 @@
         ]
       }
     ],
-    "New Architecture": ["the-new-architecture/landing-page"]
+    "New Architecture": ["the-new-architecture/landing-page"],
+    "Legacy": [
```

Comment:
"Legacy Architecture" or "Old Architecture" to stay consistent with the terminology we used in OSS

Reply:
I like that.